.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: android
+android:
+  components:
+    # travis/android: https://docs.travis-ci.com/user/languages/android/
+    - tools
+    - platform-tools
+    - tools
+    # The BuildTools version used by the project
+    - build-tools-28.0.3
+    # The SDK version used to compile the project
+    - android-28
+
+before_cache:
+  - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
+before_install:
+  - yes | sdkmanager "platforms;android-28" "platform-tools" "tools"
+cache:
+  directories:
+    - "$HOME/.gradle/caches/"
+    - "$HOME/.gradle/wrapper/"
+    - "$HOME/.android/build-cache"
+before_script:
+  - export VERSION_TO_BUILD="$TRAVIS_TAG"
+  - export PATH="$ANDROID_HOMEtools/bin:$PATH"
+script:
+  - "./gradlew --stacktrace check --no-daemon"
```

Comment:
Shouldn'd this be `test` instead of `check`?
Or does `check` already depend on `test` (not sure 🤔 )

Reply:
Check runs tests as well as it depends on the `test` task

Reply:
Sweet!